src/views/Verbose.vue

```diff
@@ -71,6 +71,37 @@
             v-model="filter"
             :placeholder="$t('verbose.filterPlaceholder')"
           >
+          <div>
+            <div
```

Comment:
This should be a loop actually.

```
<div 
  v-for="value in ['true', 'false', 'undefined']"
  :class="['exclude-result', { selected: isExcluded(value) }]"
  @click="excludeResult(value)"
>
... etc
```

Reply:
Then you can remove the repeated ones

Reply:
That is unbelievably sexy, thank you for the advice